Decks/Source/Deck.cpp

```diff
@@ -1,24 +1,67 @@
 #include "Deck.hpp"
 
+Deck::Deck()
+{
+  cards = std::make_shared<std::vector<Card>>(_cards);
+}
+
+uint Deck::findCard(Card& cardToFind)

```

Comment:
Jak zrobić to lepiej i łatwiej:

https://thispointer.com/c-how-to-find-an-element-in-vector-and-get-its-index/